### PR TITLE
CORDA-611: Improve documentation for Node Explorer

### PR DESCRIPTION
**Pull request by vkolomeyko**
Also align the name for run configuration for Node Explorer GUI

### CODE REVIEW

File: docs/source/node-explorer.rst [hunk 1]
```diff
@@ -76,6 +76,8 @@ Please note you are not allowed to login to the notary.
 .. note:: Alternatively, you may start the demo nodes from within IntelliJ using either of the run configurations
           ``Explorer - demo nodes`` or ``Explorer - demo nodes (simulation)``
 
+.. note:: It is also possible to start Explorer GUI from IntelliJ via ``Explorer - GUI`` run configuration, provided that optional TornadoFX plugin been installed first.
```
Comment (fenryka) [inline docs/source/node-explorer.rst]: to start *the* Explorer

it reads better I think with a the in it

Comment (fenryka) [reply docs/source/node-explorer.rst]: that *the* optional

Comment (fenryka) [reply docs/source/node-explorer.rst]: plugin *has* been

Comment (fenryka) [reply docs/source/node-explorer.rst]: So something like:

It is also possible to start the Explorer GUI from IntelliJ via ``Explorer - GUI`` run configuration, provided that the optional TornadoFX plugin has  been installed first.

Comment (vkolomeyko) [reply docs/source/node-explorer.rst]: Sure - will change

**Comment by vkolomeyko** [comment]
Wording updated

**Pull request merged**